pulser/simulation/simconfig.py

```diff
@@ -62,8 +68,9 @@ class SimConfig:
     epsilon: float = 0.01
     epsilon_prime: float = 0.05
     solver_options: qutip.Options = qutip.Options(max_step=5)
+    spam_dict: dict[str, float] = field(default_factory=dict)
```

Comment:
Why are you adding this one here? Is it the typing?

Reply:
Yeah, `mypy` complains that I use `self.spam_dict` otherwise, since I add it in `__post_init__`.

Reply:
Ok, let's hide it from init by setting the `field(init=False, ...)` option. And you can do the same with `doppler_sigma` while you're at it (make it's default be the value corresponding to the default temperature)